packages/api/tests/task.test.js

```diff
@@ -788,7 +779,7 @@ describe('Task tests', () => {
       });
     });
 
-    xtest('should get all tasks that are related to a farm, but not from different farms of that user', async (done) => {
+    xtest('should get all tasks related to a farm, but not from different farms of that user', async (done) => {
```

Comment:
@MwayaB  `xtest` is a way of disabling an individual `test`. This is one reason that tests show as "skipped" in the results.